### PR TITLE
6/api/reviews/review id/comments

### PR DESCRIPTION
**Pull request by ggr1992**

### CODE REVIEW

**Comment by ggr1992** [comment]
Update is merging Ticket 6 with ticket 5 after pull request was approved

File: __tests__/6-get-api-reviews-comments.test.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    })
+    })
+    test('that the comments are returned in the correct order' , () => {
+        return request(app)
+        .get("/api/reviews/3/comments")
+        .expect(200)
+        .then((result) => {
+            const comments = result.body.comments
+            expect(comments).toBeSortedBy('created_at')
+        })
```
Comment (Jodiech) [inline __tests__/6-get-api-reviews-comments.test.js:43]: This should be sorted by descending, I would recommend using jest-sorted in your tests for this. Here is the link: https://github.com/P-Copley/jest-sorted

File: models/6-reviews-comments.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const connection = require('../db/connection')
+
+exports.selectReviewByComments = (review_id) => {
+    const selectingQuery = `
+    SELECT comment_id,votes,created_at,author,body,review_id
+     FROM comments
+     WHERE review_id = $1
+     ORDER BY created_at ASC;
```
Comment (Jodiech) [inline models/6-reviews-comments.js]: Your reviews should be sorted by descending order - please change this 👍

File: __tests__/6-get-api-reviews-comments.test.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    })
+    test('throw an error if review id does not have any information', () => {
+        return request(app)
```
Comment (Jodiech) [inline __tests__/6-get-api-reviews-comments.test.js:46]: This test description isn't very semantic - "responds with a status 404 for a non existent ID.

File: __tests__/6-get-api-reviews-comments.test.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        .get("/api/reviews/10000/comments")
+        .expect(404)
+        .then((result) => {            
+            expect(result.body.msg).toBe('Not Found!')
+        })
+    })
+    test('throw an error if reviews id is a string', () => {
```
Comment (Jodiech) [inline __tests__/6-get-api-reviews-comments.test.js]: This test description doesn't match what you are testing for. This should be "responds with a status 400 for an invalid ID"

**Review by Jodiech — COMMENTED** [review]
Great work on this ticket 🚀 Please edit your tests as per the comments below and let me know if you need any help with anything 😄

**Review by Beth-McCormack — APPROVED** [review]

**Pull request merged**